lib/src/utils/widgets/linked_in_buttons.dart

```diff
@@ -11,45 +11,45 @@ class LinkedInButtonStandardWidget extends StatelessWidget {
     required this.onTap,
     this.iconHeight = 30,
     this.iconWeight = 30,
-    this.iconAssetPath = 'assets/linked_in_logo.png',
-    this.buttonText = 'Sign in with LinkedIn',
-    this.buttonColor = Colors.white,
-    this.textPadding = const EdgeInsets.all(4),
+    this.buttonText = '',
```

Comment:
this would be a breaking change as some of the users are using `Sign in with LinkedIn` text by default. We need to avoid breaking changes even if getters/methods are ugly implemented we could always cover them by tests